chingsley/ireporter
# Ft fetch patchcomment #163059894

#### What does this PR do?
This pull request deals with implementation of fetch to PATCH/EDIT the comment of a record.

#### Description of Task to be completed?
- create the editrecord.js file to implement the fetch
- using fetch consume the API with the route /api/v1/<record-type>/<record-id>/comment, setting up the request object to use the 'PATCH' method and the form data to contain the modified comment.
- Display error messages by calling the 'showDialogMsg()' function with the right arguments
- on successful fetch, display a success message.

#### How should this be manually tested?
- clone the repo
- cd into the root directory
- run `npm install' to install the necessary dependencies
- run `npm test' to see the tests pass
- run `npm start' to start the server
- ensure that you are a registered user and that you have created some records already. To edit the comment of a record, upon login, click on the edit button of the record you wish to edit, this action takes you to the 'recordEdit.html' page where you can modify the comment and click the 'save changes' button to save the new comment. Note: only records with status 'draft' can be changed. 

#### What are the relevant pivotal tracker stories
 #163059894
